tests/VolleyManagement.Domain.UnitTests/PlayerTests.cs

```diff
@@ -0,0 +1,223 @@
+﻿using System;
+using FluentAssertions;
+using VolleyManagement.Domain.PlayersAggregate;
+using Xunit;
+
+namespace VolleyManagement.Domain.UnitTests
+{
+    public class PlayerTests
+    {
+        [Fact]
+        public void Player_BirthDateHeightWeightAreNull_ExceptionIsNotThrown()
+        {
+            // Arrange
+            var player = CreatePlayer();
+
+            // Act
+            Action act = () =>
+            {
+                player.BirthYear = null;
+                player.Height = null;
+                player.Weight = null;
+            };
+
+            // Assert
+            act.Should().NotThrow<ArgumentException>("Player's birth date, height and weight can be null.");
+        }
+
+        #region FirstName
+
+        [Fact]
+        public void Player_EmptyFirstName_ExceptionIsThrown()
+        {
+            // Arrange
+            var player = CreatePlayer();
+
+            // Act
+            Action act = () => { player.FirstName = string.Empty; };
+
+            // Assert
+            act.Should().Throw<ArgumentException>("Empty first name is not allowed.");
+        }
+
+        [Fact]
+        public void Player_FirstNameDoesNotCorrespondsRegex_ExceptionIsThrown()
```

Comment:
Rename second part. You should describe requirement. (e.g. we could change code to not use regex to check it)